include/TPP/Dialect/Perf/PerfOps.td

```diff
@@ -98,6 +98,8 @@ def Perf_StopTimerOp : Perf_Op<"stop_timer", []> {
 
 def Perf_BenchOp : Perf_Op<"bench",
     [AutomaticAllocationScope, SingleBlockImplicitTerminator<"perf::YieldOp">,
+     MemoryEffects<[MemWrite, MemRead, MemAlloc<AutomaticAllocationScopeResource>]>,
```

Comment:
We don't allocate memory any more, why do we need this effect?

Reply:
After digging a bit further, good point.
`RecursiveMemoryEffects` should be enough to "derive" memory effect from nested ops.